variables.tf

```diff
@@ -1 +1,172 @@
+### REQUIRED Variables
+variable "name" {
+  description = "Name of the vault stack, will be use to prefix resources"
+  type        = "string"
+}
 
+variable "environment" {
+  description = "Type of environment -- must be one of: dev, test, prod"
+  type        = "string"
+}
+
+variable "ami_owner" {
+  description = "Account id/alias of the AMI owner"
+  type        = "string"
+}
+
+variable "key_pair_name" {
```

Comment:
recommend making this optional, if possible. likely want to restrict login methods to this system

Reply:
i don't think this can be optional without a default value?